librz/analysis/serialize_analysis.c

```diff
@@ -1928,12 +1930,11 @@ RZ_API bool rz_serialize_analysis_classes_load(RZ_NONNULL Sdb *db, RZ_NONNULL Rz
 }
 
 RZ_API void rz_serialize_analysis_types_save(RZ_NONNULL Sdb *db, RZ_NONNULL RzAnalysis *analysis) {
-	sdb_copy(analysis->sdb_types, db);
+	rz_serialize_types_save(db, analysis->typedb);
```

Comment:
I think we are now missing serialization of the noreturn database and possibly other things that have been inside `analysis->sdb_types` before, right? If so, these should be serialized too and unit tests added.

Reply:
Thanks, completely forgot about that :D

Reply:
Added in https://github.com/rizinorg/rizin/pull/896/commits/6ef63f22ce1a9294e31c5ca39d8f3e58f52516eb